### DISCUSSION TITLE
Branch names

### DISCUSSION THREAD

**Discussion by benjaminoakes** [benjaminoakes/maid#232] [General]
@coaxial Just curious: what's the pattern you use for branch names?  I understand the first part before the dash, but what's the second part?

token-rotoweed
rubocop-isheraid
etc

**Comment by coaxial** [comment]
The second part is random: ``git co -b lala-`gpw 1` ``. I'm not usually super inspired about branch names. I'll end up naming two branches the same at some point and confuse myself. This way I don't have to think too hard about branch names and never get collisions.

**Comment by benjaminoakes** [comment]
Nice! That's an interesting idea.
